src/ddmd/attrib.d

```diff
@@ -626,6 +648,35 @@ extern (C++) final class AlignDeclaration : AttribDeclaration
         return createNewScope(sc, sc.stc, sc.linkage, sc.cppmangle, sc.protection, sc.explicitProtection, this, sc.inlining);
     }
 
+    structalign_t getAlignment(Scope* sc)
```

Comment:
I believe these were deliberately turned into free functions in order to make it easier to use the compiler as a library. Please don't revert this.

Reply:
ping @ibuclaw cc @RazvanN7

Reply:
Indeed, Walter is correct, please do not revert this.